examples/src/indexes/vector_stores/rockset.ts

```diff
@@ -0,0 +1,32 @@
+import * as rockset from "@rockset/client";
+import { ChatOpenAI } from "langchain/chat_models/openai";
+import { RetrievalQAChain } from "langchain/chains";
```

Comment:
You're welcome to use this chain but we've moved towards expression language:

https://js.langchain.com/docs/expression_language/cookbook/retrieval